pvl_hsu_soiling.m

```diff
@@ -0,0 +1,351 @@
+%     are correlated with the entries in Time.
+%
+%   PM2_5 is the concentration of particulate matter with diamter less than
+%     2.5 microns, in g/m^3. Historical US concentration data sets may be
+%     found at https://aws.epa.gov/aqsweb/airdata/download_files.html. With
+%     file descriptions (for hourly data sets) at 
+%     https://aqs.epa.gov/aqsweb/airdata/FileFormats.html#_hourly_data_files
+%   
+%   PM10 is the concentration of particulate matter with diamter less than
+%     10 microns, in g/m^3. Historical US PM data sets found as described
+%     above.
+%
+%   ModelType is an optional input to the function to determine the 
+%     the model type to be used in the soiling model. A
+%     value of "1" indicates that the Variable Deposition Velocity model
+%     shall be used, a value of "2" indicates that the Fixed Settling
+%     Velocity model shall be used, and a value of "3" indicates that the
+%     Fixed Deposition Velocity model shall be used. [1] indicates that the
+%     Fixed Settling Velocity model performs best under a wide range of
+%     conditions, and thus "2" is the default ModelType if none is entered. 
+%     Validation efforts by Sandia National Laboratories
+%     confirms these findings. If an incorrect ModelType is provided, the
+%     Fixed Settling Velocity will be used (with a warning).
+%
+%   RainAccPeriod is an optional input that specifies the period, in hours
+%     over which to accumulate rainfall totals before checking against the
+%     rain cleaning threshold. For example, if the rain threshold is
+%     0.5 mm per hour, then RainThresh should be 0.5 and RainAccPeriod
+%     should be 1. If the threshold is 1 mm per hour, then the values
+%     should be 1 and 1, respectively. The minimum RainAccPeriod is 1
+%     hour. The default value is 1, indicating hourly rain accumulation.
+%     Accumulation periods exceeding 24 (daily accumulation) are not
+%     recommended.
+%   
+%   LUC is an optional input to the function, but it is required for the
+%     Variable Deposition Model. LUC is the Land Use Category as specified
+%     in Table 19.2 of [2]. LUC must be a numeric scalar with value 1, 4,
+%     6, 8, or 10, corresponding to land with evergreen trees, deciduous
+%     trees, grass, desert, or shrubs with interrupted woodlands. The LUC
+%     changes the mass deposition by Brownian diffusion. If omitted, the
+%     default value of 8 (desert) is used. Note that the value of gamma is
+%     the only thing changed by the LUC. LUC 1 and 4 use gamma = 0.54,
+%     while LUC 6, 8, and 10 use gamma = 0.56.
+%
+%   WindSpeed is an optional input to the function, but is required for the
+%     Variable Deposition Model. WindSpeed is a scalar or vector value with
+%     the same number of elements as Time, and must be in meters per
+%     second. If WindSpeed is omitted, the value of 2 m/s is used as
+%     default.
+%
+%   Temperature is an optional input to the function, but is required for
```

Comment:
Temperature is a required input if the Variable Deposition Model is selected, i.e., ModelType = 1